### ISSUE TITLE
propType warning on RawWidget.dataId

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#1601]
Got following error.
`dataId` shall be a string but in some cases it's a number but it shall be handled as a string.
Being a number is just a particular case.

```
prop type: Invalid prop `dataId` of type `number` supplied to `RawWidget`, expected `string`.
    in RawWidget (created by Connect(RawWidget))
    in Connect(RawWidget) (created by TableQuickInput)
    in form (created by TableQuickInput)
    in TableQuickInput (created by Connect(TableQuickInput))
    in Connect(TableQuickInput) (created by TableFilter)
    in div (created by TableFilter)
    in div (created by TableFilter)
    in TableFilter (created by Connect(TableFilter))
    in Connect(TableFilter) (created by Table)
    in div (created by Table)
    in div (created by Table)
    in div (created by Table)
    in div (created by Table)
    in Table (created by OnClickOutside(Table))
    in OnClickOutside(Table) (created by Connect(OnClickOutside(Table)))
    in Connect(OnClickOutside(Table)) (created by Window)
    in div (created by Tab)
    in Tab (created by Connect(Tab))
    in Connect(Tab) (created by Tabs)
    in div (created by Tabs)
    in div (created by Tabs)
    in div (created by Tabs)
    in Tabs (created by Connect(Tabs))
    in Connect(Tabs) (created by Window)
    in div (created by Window)
    in div (created by Window)
    in Window (created by MasterWindow)
    in div (created by Container)
    in div (created by Container)
    in Container (created by Connect(Container))
    in Connect(Container) (created by MasterWindow)
    in MasterWindow (created by Connect(MasterWindow))
    in Connect(MasterWindow) (created by RouterContext)
    in RouterContext (created by Router)
    in Router (created by App)
    in div (created by NotificationHandler)
    in div (created by NotificationHandler)
    in NotificationHandler (created by Connect(NotificationHandler))
    in Connect(NotificationHandler) (created by App)
    in Translation (created by App)
    in Provider (created by App)
    in ShortcutProvider (created by App)
    in App
```

**Comment by siemiatj** [comment]
That's why I added a type check. I'll find it and type cast it to string. Thanks Teo !

**Comment by siemiatj** [comment]
@metas-ts this will be fixed with #1602 as well.
